Source/GKRect.m

```diff
@@ -4,7 +4,7 @@
 #import "GKRect.h"
 
 @implementation GKRect
-@dynamic midX, midY;
+@dynamic midX, midY, maxX, maxY, origin, size;
```

Comment:
I don't think you need to use dynamic here. Dynamic is used when you're not going to supply an *implementation* for the property (because something is going to supply it dynamically at runtime - eg in core data).

That's not what's happening here is it?